toolchain/gcc.py

```diff
@@ -45,6 +45,10 @@ def setUp(self):
             packages.extend(['libmpfr-dev', 'libgmp-dev', 'libmpc-dev',
                              'texinfo', 'zip', 'libc6-dev', 'libelf1',
                              'elfutils', 'gnat', 'autogen'])
+        elif dist.name == 'debian':
```

Comment:
I see only few difference between `Ubuntu` and `debian`. Can you add debian list as base list and extend left overs like `'texinfo', 'gnat'` for Ubuntu under same condition?